src/ripple/app/misc/AMM.h

```diff
@@ -0,0 +1,160 @@
+//------------------------------------------------------------------------------
```

Comment:
Can we move this file to `xrpl_core`, so we can access the functions in clio?

Reply:
Actually, I realized we can't move everything in this file into `xrpl_core`, but we can probably move things like `calcAccountID` and `calcAMMGroupHash`